scripts/test_client

```diff
@@ -57,25 +64,19 @@ pushd user1
     [ -e app-instance ] || cp ../application_deployer/app-instance .
 
     # Submit transaction data to the aggregation server
-    zecale submit --name ${APP_NAME} --tx  ${TEST_DATA_DIR}/dummy_app/extproof1.json
-    zecale submit --name ${APP_NAME} --tx  ${TEST_DATA_DIR}/dummy_app/extproof2.json
-    zecale submit --name ${APP_NAME} --tx  ${TEST_DATA_DIR}/dummy_app/extproof3.json
-    zecale submit --name ${APP_NAME} --tx  ${TEST_DATA_DIR}/dummy_app/extproof4.json
-
-    echo Check starting app state
-    zecale_dummy_app get 7 --check 0
-    zecale_dummy_app get 8 --check 0
-    zecale_dummy_app get 9 --check 0
-    zecale_dummy_app get 10 --check 0
+    zecale submit ${TEST_DATA_DIR}/dummy_app/extproof1.json
+    zecale submit ${TEST_DATA_DIR}/dummy_app/extproof2.json
+    zecale submit ${TEST_DATA_DIR}/dummy_app/extproof3.json
+    zecale submit ${TEST_DATA_DIR}/dummy_app/extproof4.json
 
     # Get a single batch and send to the contract
     zecale get-batch --name ${APP_NAME} --batch-file batch1.json
     zecale check-batch batch1.json --batch-size 2
     zecale submit-batch batch1.json --wait
 
     echo Check app state after batch 1
-    zecale_dummy_app get 7 --check 1
-    zecale_dummy_app get 8 --check 1
+    zecale_dummy_app get 7 --check 23
```

Comment:
Hum.. isn't the value of the flag `--check` supposed to be {0,1}? (see the flag documentation pasted below)
```python
"--check", type=int, help="Check the result against the given value (0|1)")
```

Reply:
I've updated the help text.
(The type as been switched to `int`, so it no longer needs to be `0|1`.  This is how we check that the extra parameters for each nested tx get propagated to the client application.)